tron/mcp.py

```diff
@@ -150,15 +163,15 @@ def get_job_collection(self):
     def get_config_manager(self):
         return self.config
 
-    def restore_state(self, action_runner):
+    def restore_state(self, action_runner, tron_start_time=None):
         """Use the state manager to retrieve to persisted state and apply it
         to the configured Jobs.
         """
-        log.info("restoring")
+        log.info("restoring from DynamoDB")
```

Comment:
```suggestion
        log.info("Restoring from DynamoDB")
```
would be more consistent with the other logs we're adding (sentence cased)